Readme.md

```diff
@@ -71,14 +123,30 @@ write_chunk(
 
 The written file can now be read in Python via
 
+### Polars
 ```python
-# using polars
 import polars as pl
 pl.read_parquet("example.pq")
+shape: (3, 2)
```

Comment:
If you include the content, I would prefer to use `>>>` to mark the code, as in:

```python
>>> import polars as pl
>>> pl.read_parquet("example.pq")
shape: (3, 2)
┌─────┬─────┐
│ a   ┆ b   │
│ --- ┆ --- │
│ f32 ┆ i32 │
╞═════╪═════╡
│ 1.0 ┆ 1   │
│ 2.0 ┆ 2   │
│ 3.0 ┆ 3   │
└─────┴─────┘
```

Reply:
DOne. BTW I think including the content is valuable so it is clear to people the struct is not serialized as a single `StructArray` column but as individual fields